src/main/java/com/google/devtools/build/lib/skyframe/ProcessPackageDirectory.java

```diff
@@ -125,14 +125,18 @@ public ProcessPackageDirectoryResult getPackageExistenceAndSubdirDeps(
 
     if (packageId.getRepository().isMain()
         && fileValue.isSymlink()
-        && fileValue
+        && (fileValue
             .getUnresolvedLinkTarget()
-            .startsWith(directories.getExecRootBase().asFragment())) {
+            .startsWith(directories.getExecRootBase().asFragment())
+            || rootRelativePath.getBaseName().startsWith("bazel-"))) {
```

Comment:
I need to get the actual symlink prefix here rather than hard-coding it as `bazel`. However, I have no idea how to get this value here. Can somebody help?